ftw/contentnav/behaviors/content_categories.py

```diff
@@ -66,6 +66,13 @@ def new_content_categories(self, value):
             return ()
         else:
             categories = tuple(self.annotation_storage.content_categories)
+
+            # Strip all new categories and remove duplicates.
```

Comment:
No worries ☕️ By the way, I would rather keep the variable because:
1) local consistency: `categories` is not used a second time
2) readability: all those list comprehensions, casting to tuples, sets, and tuples again makes the code hard to read.